man/sd_bus_set_address.xml

```diff
@@ -104,6 +111,11 @@
     automatically by
     <citerefentry><refentrytitle>sd_bus_open</refentrytitle><manvolnum>3</manvolnum></citerefentry> and
     similar calls, based on environment variables or built-in defaults.</para>
+
+    <para><function>sd_bus_set_exec</function> is a shorthand function for setting a
+    <literal>uniexec</literal> address that spawns the given executable with the given arguments.
+    if <parameter>argv</parameter> is <constant>NULL</constant>, the given executable is spawned
+    without any arguments.</para>
```

Comment:
Hmm, "arguments" can mean two things here: either anything after argv[0], or full argv. As written, this could be taken to mean that argv is empty, even though argv[0] is always provided. So maybe something like this:
> If `<parameter>argv</parameter>` is `<constant>NULL</constant>`, the given executable is spawned with an argument array consisting of the executable name and no other arguments.

Reply:
I'm torn but I feel since we're specifically talking about spawning an executable here that it's sufficiently clear that we're not including argv[0]. How about I change it to: 

> If <parameter>argv</parameter> is <constant>NULL</constant>, the given executable is spawned without any extra arguments.